src/uncrustify.cpp

```diff
@@ -2052,6 +2104,7 @@ static lang_name_t language_names[] =
    { "PAWN", LANG_PAWN          },
    { "OC",   LANG_OC            },
    { "OC+",  LANG_OC | LANG_CPP },
+   { "CS+",  LANG_CS | LANG_CPP },
```

Comment:
Is this intended to be in this PR?

Reply:
You are right. Should not be part of this PR. It is a special c++/c# language used at Unity.